common/src/main/resources/runtime.properties

```diff
@@ -37,6 +37,12 @@
 # If true, Falcon skips oozie dryrun while scheduling entities.
 *.falcon.skip.dryrun=false
 
+# Applicable only if Falcon is configured to use Oozie as it's scheduler
+# Minimum value that the Falcon Oozie Coordinator Builder will use when 
+# configuring the throttle for an Oozie coordinator. It is recommended that the
+# minimum be greater than or equal to the Oozie default of 12.
+*.falcon.oozie.coordinator.throttle.min=12
```

Comment:
DONE

Merged with the upstream branch. Removed my edits to CHANGES.txt. Added the property to the proper location for build.